### PR TITLE
DOC-9811 Fix admonition gap in lists

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

**Comment by maria-robobug** [comment]
@RayOffiah nice! 
Looking at this test: https://deploy-preview-135--cb-docs-ui.netlify.app/#vbucket-stats
Seems like there's still a gap above the admonition, not sure if the fact it's in a table causes it to not work?
Maybe needs further testing.

**Review by osfameron — COMMENTED** [review]
Not sure if the comment is needed (as `git blame` would point at it)?

Otherwise looks good, CSS variables oh my!

File: src/css/doc.css [hunk 1]
```diff
@@ -147,6 +147,16 @@
   margin-top: var(--base-large-space);
 }
 
+/**
```
Comment (maria-robobug) [inline src/css/doc.css]: Don't think we need two `*` for multi line comment, just one no?
```suggestion
/*
```

Comment (RayOffiah) [reply src/css/doc.css]: Mmm. Seems to be some sort of IntelliJ preference. Yup; it's fine with just the single asterisk.

**Comment by RayOffiah** [comment]
@maria-robobug 

Yes, this was specifically for lists, not tables, because I hadn't noticed that tables have the same problem. :-)

I'll see if I can fix it for table entries too.

**Comment by maria-robobug** [comment]
> Yes, this was specifically for lists, not tables, because I hadn't noticed that tables have the same problem. :-)
> I'll see if I can fix it for table entries too.

Yeah, if it's a simple fix add it. 
If not, we can just merge this for the list items and leave the table entries for another time.

Happy with whichever approach you choose 👍 .

**Review by maria-robobug — APPROVED** [review]

**Pull request merged**